### PR TITLE
Removed the licensing article from generic space

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Removed the licensing article from generic space

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

Anytime

### CODE REVIEW

**Comment by sofietoft** [comment]
Looks great 💪 
Perhaps we should put a redirect in place as well?
Just to the landing page of the generic space, perhaps?

**Comment by eshanrnh** [comment]
Makes sense. I added a redirect.

File: generic/.gitbook.yaml [hunk 1]
```diff
@@ -3,3 +3,6 @@ root: ./
 ​structure:  
     readme: README.md
     summary: SUMMARY.md
+
+redirects:
+    generic/the-licensing-model: generic/README.md
```
Comment (sofietoft) [inline generic/.gitbook.yaml:8]: I don't think the site name is required, as this file is already into the generic folder.

```suggestion
    the-licensing-model: README.md
```

**Comment by eshanrnh** [comment]
@sofietoft I think this PR is ready to be merged 😀

**Review by sofietoft — APPROVED** [review]
LGTM 💪

**Pull request merged**